testerum-backend/testerum-runner/testerum-runner-api/src/main/kotlin/com/testerum/runner/cmdline/report_type/builder/impl/JsonEventsReportTypeBuilder.kt

```diff
@@ -7,12 +7,17 @@ import java.nio.file.Path
 
 class JsonEventsReportTypeBuilder {
 
+    var reportServerURL: String? = null
```

Comment:
This property should be added to it's own class, since "reportServerURL" is not a property of the JSON_EVENTS report type. Suggested name: ``RemoteServerReportTypeBuilder``.

Also add a method for this new class in ``ReportTypeBuilders``.

Reply:
Please check if my implementation is fine